test/createAzureResource.test.ts

```diff
@@ -59,26 +59,83 @@ suite('Create Azure Resources', async function (this: ISuiteCallbackContext): Pr
         ext.azureAccountTreeItem.dispose();
     });
 
-    test('createFunctionApp (Basic)', async () => {
+    test('Create windows function app (Basic) and deploy JavaScript project', async () => {
+        const functionName: string = 'HttpTrigger';
         resourceGroupsToDelete.push(resourceName1);
-        await runWithFuncSetting('advancedCreation', undefined, async () => {
-            ext.ui = new TestUserInput([resourceName1]);
-            await vscode.commands.executeCommand('azureFunctions.createFunctionApp');
-            const createdApp: WebSiteManagementModels.Site = await webSiteClient.webApps.get(resourceName1, resourceName1);
-            assert.ok(createdApp);
+        await runWithFuncSetting('projectLanguage', ProjectLanguage.JavaScript, async () => {
+            await runWithFuncSetting('advancedCreation', undefined, async () => {
+                ext.ui = new TestUserInput([resourceName1]);
+                await vscode.commands.executeCommand('azureFunctions.createFunctionApp');
+                const createdApp: WebSiteManagementModels.Site = await webSiteClient.webApps.get(resourceName1, resourceName1);
+                assert.ok(createdApp);
+                await runWithFuncSetting('projectRuntime', ProjectRuntime.v2, async () => {
+                    const templateId: string = 'HttpTrigger-JavaScript';
+                    const authLevel: string = 'function';
+                    ext.ui = new TestUserInput([DialogResponses.skipForNow.title]);
+                    await vscode.commands.executeCommand('azureFunctions.createNewProject', projectPath, ProjectLanguage.JavaScript, ProjectRuntime.v2, false, templateId, functionName, { AuthLevel: authLevel });
+                    await validateProject(projectPath, getJavaScriptValidateOptions(true));
+                });
+            });
         });
+
+        ext.ui = new TestUserInput([resourceName1, 'Deploy']);
+        await vscode.commands.executeCommand('azureFunctions.deploy');
+        await delay(500);
+
+        // Verify the deployment result through triggerUrl
+        const getFunction: WebSiteManagementModels.FunctionEnvelopeCollection = await webSiteClient.webApps.listFunctions(resourceName1, resourceName1);
```

Comment:
It looks like you and Nathan have gone back and forth on this, but I think it's better to use `'azureFunctions.copyFunctionUrl'` to get the url. It seems simpler and I think it tests more of the real-life scenario for a user. That being said, I agree with Nathan that we don't want you to install "clipboardy" so just use the `vscode.env.clipboard` apis like this:
```
await vscode.env.clipboard.writeText(''); // clear the clipboard
ext.ui = new TestUserInput(...);
await vscode.commands.executeCommand('azureFunctions.copyFunctionUrl');
const triggerUrl = await vscode.env.clipboard.readText();
```

@nturinski are you fine with that? I feel like we might've discussed this offline, but I don't remember exactly what we talked about.

Reply:
@EricJizbaMSFT  I'm good with that.  I think what we talked about was in the context of the Storage extension, but basically using copy was fine, but we don't need a test dedicated to the copy commands.